examples/cl03.rs

```diff
@@ -151,7 +160,8 @@ fn main() {
 
     let args: Vec<String> = env::args().collect();
 
-    if args.len() != 2 {
```

Comment:
2 is correct

Reply:
Yes, my bad, forgot this was in. That was because of a discrepancy with IDE run configuration.

Reply:
No problem